docs/sources/articles/networking.md

```diff
@@ -324,6 +324,15 @@ page for further details.
 > hostname, which Docker will not recognize in the context of the
 > `--link=` option.
 
+> **Note**:
+> For `--icc=false` option to work, the bridge's traffic must be passed to
+> iptables' chains. This is controled by kernel parameters
+> `net.bridge.bridge-nf-call-iptables` and
+> `net.bridge.bridge-nf-call-ip6tables`. If set to 1, traffic will be processed
+> with `iptables` rules. However, several distributions set them to 0 by
+> default. Docker checks these parameters upon its initialization and will
+> refuse to start unless enabled if `--icc=false` is requested.
```

Comment:
unless they are enabled? if the \`--icc-false\` option is requested. 

Although the whole sentence could do with a rewrite.